Makefile

```diff
@@ -23,8 +23,10 @@ lint: $(CONFIG)
 	bundle exec reek
 	@echo "--- fasterer ---"
 	bundle exec fasterer
+	@echo "--- eslint ---"
+	eslint app
```

Comment:
On my machine I had to specify the path to the executable specifically since I did not install `eslint` globally:

```
node_modules/.bin/eslint app
```